src/Routes/TrackedRoute.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Route} from 'react-router-dom';
+import {AmplitudeProxy} from '../analyticsTracking';
+
+/**
+ * Wrap all of our routes to make them log the view and route they render
+ *
+ *  @event_scehma analyticsTracking/event_schemas/page/page_view.schema.json
+ * */
+const TrackedRoute = ({
+  component,
+  render,
+  logPageView = false,
+  eventProperties,
+  ...rest
+}) => (
+  <AmplitudeProxy
```

Comment:
You'll probably want to pass down the inherited properties here.

Reply:
`eventProperties` is spread into the AmplitudeProxy eventProperites, are you talking about passing the `...rest` to AmplitudeProxy as well?